modin/core/storage_formats/pandas/parsers.py

```diff
@@ -140,6 +141,7 @@ def find_common_type_cat(types):
 class PandasParser(ClassLogger):
     """Base class for parser classes with pandas storage format."""
 
+    @disable_logging
```

Comment:
why?

Reply:
I don't think this needs to be logged, will remove.